Source/Tests/TrackableEntities.EF.5.Tests/FamilyDbContextTests.cs

```diff
@@ -215,5 +215,21 @@ public void Apply_Changes_Should_Mark_Grandchild_Deleted()
             Assert.Equal(EntityState.Deleted, context.Entry(child).State);
             Assert.Equal(EntityState.Deleted, context.Entry(grandchild).State);
         }
+
+        [Fact]
+        public void Apply_Changes_Should_Mark_ComplexType_Modified()
+        {
+            // Arrange
+            var context = TestsHelper.CreateFamilyDbContext(CreateFamilyDbOptions);
+            var parent = new MockFamily().Parent;              
+            parent.Address.StreetName = "123 Lee Ave.";
+            parent.Address.ModifiedProperties = new[] { nameof(Address.StreetName) };
```

Comment:
I think with CTs, we don't really care about the state of the CT, but rather about its parent state, no?